src/features/crops/components/MarketItems.tsx

```diff
@@ -49,7 +49,7 @@ export const MarketItems: React.FC<Props> = ({ onClose }) => {
         <img
           src={close}
           className="h-6 cursor-pointer mr-2 mb-1"
-          onClick={onClose}
+          onClick={() => (tourIsOpen ? null : onClose)}
```

Comment:
Same suggestion as above. Also this currently would have issues since we are not invocating `onClose` -> `onClose()`